packaging/conda/meta.yaml

```diff
@@ -1,6 +1,8 @@
+{% set version = os.environ.get('KHIOPS_VERSION') %}
```

Comment:
We can write directly:

```
{% set version = os.environ.get('KHIOPS_VERSION').replace("-", "") %}
```

Reply:
I would replace "-" with "" to remain consistent with the current versioning scheme.

Reply:
done